docs/_layouts/sub-navigation.njk

```diff
@@ -3,6 +3,10 @@
 {% block scripts %}
   <script src="/assets/iframeResizer.js"></script>
   <script>
-    iFrameResize({}, `[data-module="app-example-frame"]`)
+    iFrameResize({}, `[data-module="x-govuk-example-frame"]`)
```

Comment:
The requirement to add the iFrame script to both the source content and the page that consumes the component either needs incorporating into the component somehow, else documenting how to set up. Part of me wonders if their might be something cleaver we could do with web components, but that’d be very different to how other components work.

Reply:
Yeah this is super annoying. Seems odd that there’s no way to set an iframe to natively resize to its contents but I guess there were performance reasons why that wasn’t built in?

Could bundle the iframeresizer script with the `xGovukExample` component javascript itself, but that means an extra dependency and might be tricky? Possibly worth investigating still though. Otherwise we’d just have to document it well as something extra that you have to add to make dynamic iframe sizing work. Possibly there’s some contexts where a fixed height would be ok and you wouldn’t need it, but it does feel quite essential?